lib/core.js

```diff
@@ -555,10 +557,13 @@ Archiver.prototype.append = function(source, data) {
  * @return {this}
  */
 Archiver.prototype.bulk = function(mappings) {
-  if (typeof process !== 'undefined' && typeof process.emitWarning !== 'undefined') {
-    process.emitWarning('Archiver.bulk() deprecated since 0.21.0', 'DeprecationWarning');
-  } else if (typeof console !== 'undefined' && typeof console.warn !== 'undefined') {
-    console.warn('Archiver.bulk() deprecated since 0.21.0');
+  if (!this._loggedBulkDeprecation) {
+    this._loggedBulkDeprecation = true;
+    if (typeof process !== 'undefined' && typeof process.emitWarning !== 'undefined') {
+      process.emitWarning('Archiver.bulk() deprecated since 0.21.0', 'DeprecationWarning');
+    } else {
+      console.warn('Archiver.bulk() deprecated since 0.21.0');
+    }
```

Comment:
Okay, that can be done as well. Do you want me to leave the `_loggedBulkDeprecation` in there or do you want me to log on each call?